src/components/header-block/header-block.stories.tsx

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
-import HeaderBlock from './header-block';
+import { HeaderBlock } from './header-block';
 
 export default {
   component: HeaderBlock,
   title: 'Components/HeaderBlock',
 } as ComponentMeta<typeof HeaderBlock>;
```

Comment:
You need to remove this entire `default` export for the `mdx` to have an effect.

Reply:
It is kind of weird though, since I saw the changes locally in the storybook, but in netlify they have not been visible.
But now it works everywhere :+1: